docs/internals.md

```diff
@@ -15,41 +15,21 @@ diagram illustrates the states.
 The Init state (class `BeaconSendingInitState`) sends the initial status request to the server.
 The request is retried several times (by default 5 retries, in total 6 requests) with increasing
 delays between consecutive retries.  
-If the server returned a status response a state transition to TimeSync is performed, otherwise
-OpenKit stays in the Initialize state, but sleeps some time until the next status request is sent. 
+If the server returned a successful status response a state transition to either CaptureOn or CaptureOff
+is performed, depending on whether capturing is enabled or disabled in the initial status response.
+If the status response is unsuccessful OpenKit stays in the Initialize state,
```

Comment:
Change to: "If the status request fails OpenKit stays in the Initialize state and sleeps some time until the next status request is sent."